src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -85,24 +98,47 @@ function MetadataSidebarRedesign({
         setIsUnsavedChangesModalOpen(true);
     };
 
-    const handleTemplateSelect = (selectedTemplate: MetadataTemplateInstance) => {
+    const convertToInstance = (template: MetadataTemplate): MetadataTemplateInstance => {
```

Comment:
wondering if this function should be a part of a component. Potentially it could be useful in other places + would be easier to test, if declared outside of the component

Reply:
Ok, I extracted this function to the utils folder and added a test for it.